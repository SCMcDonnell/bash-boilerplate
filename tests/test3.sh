#!/bin/bash

source ../boilerplate.sh

s1="Bash"
s2="Bash Boilerplate"
s3="And Now for Something Completely Different"

to_lower "$s1" l1
to_lower "$s2" l2
to_lower "$s3" l3

to_upper "$s1" u1
to_upper "$s2" u2
to_upper "$s3" u3

echo $l1
echo $l2
echo $l3

echo $u1
echo $u2
echo $u3
