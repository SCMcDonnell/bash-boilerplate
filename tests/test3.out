bash
bash boilerplate
and now for something completely different
BASH
BASH BOILERPLATE
AND NOW FOR SOMETHING COMPLETELY DIFFERENT
